binds/python/bindings_utils.cpp

```diff
@@ -65,3 +65,24 @@ static morphio::Points array_to_points(py::array_t<float> &buf){
     }
     return points;
 }
+
+/**
+ * @brief "Casts" a Cpp sequence to a python array (no memory copies)
+ *  Python capsule handles void pointers to objects and makes sure
+ *      that they will remain alive.
+ *
+ *      https://github.com/pybind/pybind11/issues/1042#issuecomment-325941022
+ */
+template <typename Sequence>
+inline py::array_t<typename Sequence::value_type> as_pyarray(Sequence&& seq) {
+    // Move entire object to heap. Memory handled via Python capsule
+    Sequence* seq_ptr = new Sequence(std::move(seq));
```

Comment:
can you add `#include <utility>  //for std::move`, I like files to include their deps when possible

Reply:
sure, thanks